ui/app/components/pages/confirm-transaction-switch/confirm-transaction-switch.container.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux'
+import ConfirmTransactionSwitch from './confirm-transaction-switch.component'
+
+const mapStateToProps = state => {
+  const { confirmTransaction } = state
```

Comment:
Perhaps we should map the specific properties within `confirmTransaction` to props here. It will simplify the logic within the component.